.github/drafts/16506/16506_widgets_landing_outline.md

```diff
@@ -0,0 +1,35 @@
+<!--- This page extends https://docs.appsmith.com/reference/widgets , adding a list of properties that many widgets have in common. -->
+
+## Common Properties
```

Comment:
This looks good. Just confirming that it will be added to the [widget landing page](https://docs.appsmith.com/reference/widgets).

Reply:
Yep! That is the correct page, just adding a new subheading with common properties